app/PharBuilder.php

```diff
@@ -353,7 +353,7 @@ public function buildPhar()
 
         $this->stubFile = $this->makePathRelative($this->stubFile);
         $this->phar->setStub(
-            (!$this->isSkipShebang() ? '#!/usr/bin/env php' . PHP_EOL : '') .
+            (!$this->isSkipShebang() ? '#!/usr/bin/env php' . "\n" : '') .
```

Comment:
If you launch script as a regular binary file (e.g. `bin/script`) and it has `\r` in the first line, bash fails to run it:
```
$ ./test.bash 
bash: ./test.bash: /usr/bin/bash^M: плохой интерпретатор: Нет такого файла или каталога
```